nebula/nebula.cpp

```diff
@@ -4,15 +4,8 @@
 
 #include "nebula.h"
 
-constexpr auto QRC_ROOT = "qrc:///nebula/";
+constexpr auto QRC_ROOT = ":/nebula";
 
 void Nebula::Initialize(QQmlEngine* engine) {
-  Q_INIT_RESOURCE(components);
-  Q_INIT_RESOURCE(themes);
-  Q_INIT_RESOURCE(nebula_resources);
-  Q_INIT_RESOURCE(compatQt6);
-  Q_INIT_RESOURCE(resourcesQt6);
-  Q_INIT_RESOURCE(utils);
-
   engine->addImportPath(QRC_ROOT);
```

Comment:
What's the impact of removing `addImportPath()`? It would be nice if we could just do away with the need to initialize the nebula library entirely.